src/busy-indicator.scss

```diff
@@ -79,6 +88,16 @@ $fd-dots-size-l: 2rem;
   height: $fd-dots-size-l;
 }
 
+.#{$block}--label {
```

Comment:
This is not a modifier class. You can't have `fd-busy-indicator--label`. You need `fd-busy-indicator__label`. You then need to include the reset function.

Reply:
fixed